frontend/src/router.js

```diff
@@ -7,6 +7,12 @@ import { getEnv } from '@/environment.js'
 
 Vue.use(Router)
 
+slugify.extend({
+  '@': '(at)',
+  '.': ' ',
```

Comment:
Are spaces really a good character for use in an URL? And is this the correct place to extend slugify? Shouldn't we rather create a central place (like the ones we call plugins so far) where slugify is imported and configured, and then everywhere in the app we import from there?

Reply:
[aaa9a17](https://github.com/ecamp/ecamp3/pull/3797/commits/aaa9a1762a0582c98e2d8738339fb3f76374c07f)